internal/cli/auth/whoami.go

```diff
@@ -35,7 +35,7 @@ func (opts *whoOpts) Run() error {
 	return nil
 }
 
-var ErrUnauthenticated = errors.New("not logged in")
+var ErrUnauthenticated = errors.New("not logged in with an Atlas account")
 
 func AccountWithAccessToken() (string, error) {
```

Comment:
what about
```suggestion
func AccountWithAccessToken() (string, error) {
	if config.PublicAPIKey() != "" {
		return config.PublicAPIKey(), nil
	}
	if config.AccessToken() != "" {
		return config.AccessTokenSubject()
	}
	return  "", ErrUnauthenticated
}
```

Reply:
if we are doing this shouldn't we say `Logged in as email@server.com` and `Logged in as public key XXX` as opposed to `Logged in as XXX`

Reply:
```suggestion
func AccountWithAccessToken() (string, string, error) {
	if config.PublicAPIKey() != "" {
		return config.PublicAPIKey(), "key", nil
	}
	if config.AccessToken() != "" {
		return config.AccessTokenSubject(), "account"
	}
	return  "", "", ErrUnauthenticated
}
```

Reply:
makes sense to add a bit more context as I also realized that AccountWithAccessToken() is re-used in the setup cmd so I split the logic too